about_regular_expressions.exs

```diff
@@ -0,0 +1,36 @@
+defmodule AboutRegularExpressions do
+  use Koans
+
+  think "A simple regular expression" do
+    regex = Regex.compile!("This [W|w]orks!")
```

Comment:
Yeah, I'm kind of torn about using the sigil right away or not. I would imagine that a beginner not coming from ruby wouldn't have a hard time understanding sigils and especially delimiters. That's why I started out with "here's a string, stuff it into a function and you get a regex".